static/js/chart/draw_scatter.ts

```diff
@@ -757,13 +757,14 @@ export function drawScatter(
     yPerCapita: boolean
   ) => JSX.Element
 ): void {
+  const container = d3.select(svgContainerRef.current);
+  container.selectAll("*").remove();
```

Comment:
what is this change for?

Reply:
since we redraw when the window resizes, we need to remove the last drawn chart first. We do this for all our other charts too